plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/image/block.json

```diff
@@ -24,6 +24,7 @@
 			"clientNavigation": true
 		},
 		"html": false,
+		"layout": true,
```

Comment:
This may be leftover or kept on purpose. It adds `Inner blocks use content width` option but please confirm if you wanted to add it or it should be removed.